data/campaigns/Eastern_Invasion/_main.cfg

```diff
@@ -31,28 +112,27 @@
         [/entry]
     [/about]
     [about]
-        title = _ "Campaign Maintenance"
-        [entry]
-            name = "Dimitar Ilccov (Mythological)"
-        [/entry]
-        [entry]
-            name = "Lari Nieminen (zookeeper)"
-            comment = "current maintainer"
-        [/entry]
+        title = _ "Campaign Redesign"
         [entry]
-            name = "Astrid Halberkamp"
+            name = "Dalas"
         [/entry]
     [/about]
     [about]
-        title = _ "Campaign Epilogue and Continuity"
+        title = _ "Prose and Story Edits"
         [entry]
-            name = "Eric S. Raymond (ESR)"
+            name = "nemaara"
         [/entry]
     [/about]
     [about]
-        title = _ "Prose and Story Edits"
+        title = _ "Campaign Maintenance"
```

Comment:
Should probably be renamed "Campaign Maintainers Emeritii" or "Previous Campaign Maintainers" or something if they're no longer involved